locale/hi.yml

```diff
@@ -1286,8 +1286,8 @@ yield: "उपज" # ✅
 
 # << Built-In Functions >>
 
-abs: "एब्स" # ❓ "परम"
-all: "सभी" # ❓ "सब"
+abs: "परम" # ❓ "परम"
+all: "सब" # ❓ "सब"
```

Comment:
For these, keep the initial translation and add your review at the end of the last user's review ^ That way when more reviewers contribute to the translation, we're able to see all of the previous versions and be able to come to a fuller consensus. 👍

Reply:
Okay then, I willl add my review at the end.

Reply:
I apologize for the confusion. Thanks so much for going through with another revision, but I don't believe I was clear enough in my change request. 

It is important to keep all of the iterations of review in tact, so we're able to refer to them for the final review. **Line 1289-1290** would look something like this:

```yml
abs: "एब्स" # ❓ "परम", @sshiv5768 "परम"
all: "सभी" # ❓ "सब", @sshiv5768 "सब"
```

For something like **line 1299** where the English keyword is still in place (`chr: "chr"`), this would be acceptable:
```yml
chr: "चर" # ❓ "अक्षर" I believe data types should be kept as it is just change them phonetically if needed, @sshiv5768 ✅
```

Also, if you agree with the translation how it is, add a ✅  so we know its great! So like for **line 1297**:
```yml
bytes: "बाइट्स" # ✅✅
```


_You don't have to add your name by each of your translation reviews, but it can be helpful if you leave a comment/explanation as to why you feel a word better describes the keyword or you have additional ideas._

Hope this helps clarify my request! Please feel free to ask my anything to better explain any aspect of this^